swarm/storage/localstore/gc.go

```diff
@@ -140,10 +140,11 @@ func (db *DB) incGCSize(count int64) {
 
 var writeGCSizeDelay = 10 * time.Second
```

Comment:
Reasons:
- var instead const: it may be required by some test to change this value, currently not
- outside of function: literal in time.After would degrade the significance of the value which is now described by variable name